### ISSUE TITLE
jpeg ???

### ISSUE THREAD

**Issue by jfhauris** [PingoLH/FCHarDNet#59]
The FCHardnet operates on png images.  Will it work on jpeg images?  Jon

**Comment by riazspace** [comment]
@jfhauris Yes it works on JPEG images. but you need to modify the inference code.
